reference/rangefinder.md

```diff
@@ -125,6 +125,8 @@ int wb_range_finder_get_sampling_period(WbDeviceTag tag)
 
 `wb_range_finder_enable()` allows the user to enable a range-finder update each
 `ms` milliseconds.
+The provided `ms` argument specifies the sensor's sampling period.
```

Comment:
If we want to change it, then it is better to use the same terminology everywhere, so using ``sampling_period``instead of ``refresh_rate`.